### ISSUE TITLE
CC0091 wrongly tries to make static methods in structs

### ISSUE THREAD

**Issue by jakubsuchybio** [code-cracker/code-cracker#681]
Hi.
Another CC0091 bug. It tries to change method in struct that does not only manipulate with static fields.
Code proof:
```c#
		struct Foo
		{
			private int _x, _y;

			public void Goo(int x, int y)
			{
				_x = x;
				_y = y;
			}
		}
```
Goo is here wrongly under warning of CC0091.

**Comment by giggio** [comment]
Bug confirmed.

**Comment by jakubsuchybio** [comment]
Fix confirmed with 1.0.0-rc5-1063 on our codebase. Thanks! Btw is it ok if I write this? Or are your tests enough? :)

**Comment by giggio** [comment]
It is ok, and you actually help us to make sure. Thanks!
